dpiprobe.go

```diff
@@ -27,6 +27,7 @@ func main() {
 	disableIpPtrLookup := flag.Bool("n", false, "Disable IP PTR lookup.")
 	timeoutSeconds := flag.Uint("t", 15, "Timeout for each hop.")
 	port := flag.Uint("port", 0, "Port number.")
+	serverName := flag.String("host", "", "SNI in ClientHello packet")
```

Comment:
Please adjust the message to indicate that this also has an effect on HTTP mode (sets the Host HTTP header).